sources/packages/api/src/services/user/user.service.ts

```diff
@@ -14,4 +15,28 @@ export class UserService extends DataModelService<User> {
       userName,
     });
   }
+
+  async synchronizeUserInfo(userInfoBCServiceCard: UserInfo): Promise<User> {
+    let userToSync = await this.getUser(userInfoBCServiceCard.userName);
+
+    //Initializing a new user if one is not found
+    if (!userToSync) {
+        throw new Error(
+          `Not able to find a user using the username (bcsc name)`
```

Comment:
Consider do not use the string interpolation syntax if there is no variable to be added in the text.

Reply:
@andrewsignori-aot Incase of multiline string or string with "/', we can use ``.